src/Options/ServerOptions.php

```diff
@@ -92,7 +92,7 @@ public function __construct($options = null)
      *
      * @param  array $options
      */
-    public function setFromArray(array $options)
+    protected function setFromArray(array $options)
```

Comment:
okey, ... how do you ensure passed values are of the correct type?

```
setRevokeRotatedRefreshTokens(bool $revokeRotatedRefreshTokens)
```
only accepts bool